hack/multi-node/bootkube-up

```diff
@@ -25,7 +25,7 @@ if [ ! -d "cluster" ]; then
   ../../_output/bin/${local_os}/bootkube render --asset-dir=cluster --api-servers=https://172.17.4.101:6443 ${etcd_render_flags} ${cnp_render_flags}
   cp user-data.sample cluster/user-data-worker
   cp user-data.sample cluster/user-data-controller
-  sed -i -e '/node-role.kubernetes.io\/master/d' cluster/user-data-worker
+  sed -i -e '/node.kubernetes.io\/master/d' cluster/user-data-worker
```

Comment:
We shouldn't do this. The `node-role.kubernetes.io` label should be used since `node-role.kubernetes.io/master` is "special" for security reasons. The `node.kubernetes.io` label used for node status/conditions. I know this puts the burden of labeling the joining master nodes on the user, but I don't we can do any better than that without a much bigger change.

Reply:
so since coreos uses the kubelet to self lebel/taint itself this is no longer possible with the NodeRestriction, im open to suggestions
* How about i leave the `node-role` in and have it removed in the hack/* so that `node-role` remains in render and the hack/* can do what it requires.

Reply:
Based on https://github.com/kubernetes/kubernetes/issues/75457#issuecomment-532609381, we will keep the `node-role` in all the templates and its upto the tooling with uses bootkube to manage the labels, so the `hack/*` will handle the label wrt to this repo